### PR TITLE
show ToS error in connect manual bank account page

### PR DESCRIPTION
**Pull request by adeel0202**
### Details
Show the error when the user doesn't accept Terms of Service on Connect bank account manually page

### Fixed Issues
$ #7839

### Tests
- [x] Verify that no errors appear in the JS console

#### Contributor (PR Author) Checklist
- [x] I made sure to pull `main` before submitting my PR for review
- [x] I linked the correct issue in the `### Fixed Issues` section above
- [x] I wrote clear testing steps that cover the changes made in this PR
    - [x] I clearly indicated the environment tests should be run in (Staging vs Production)
- [ ] I wrote testing steps that cover success & fail scenarios (if applicable)
- [ ] I ran the tests & they passed on **all platforms**
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I verified there are no console errors related to changes in this PR
- [x] I followed proper code patterns (see [Reviewing the code](https://github.com/Expensify/App/blob/main/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [ ] I added comments when the code was not self explanatory
    - [ ] I put all copy / text shown in the product in all `src/languages/*` files (if applicable)
    - [ ] I followed proper naming convention for platform-specific files (if applicable)
    - [x] I followed style guidelines (in [`Styling.md`](https://github.com/Expensify/App/blob/main/STYLING.md)) for all style edits I made
- [ ] I followed the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/PR_REVIEW_GUIDELINES.md)

#### PR Reviewer Checklist
- [x] I verified the Author pulled `main` before submitting the PR
- [x] I verified the correct issue is linked in the `### Fixed Issues` section above
- [x] I verified testing steps are clear and they cover the changes made in this PR
    - [x] I verified the testing environment is mentioned in the test steps
- [x] I verified testing steps cover success & fail scenarios (if applicable)
- [x] I verified tests pass on **all platforms** & I tested again on all platforms
- [x] I checked that screenshots or videos are included for tests on [all platforms](https://github.com/Expensify/App/blob/main/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I verified there are no console errors related to changes in this PR
- [x] I verified proper code patterns were followed (see [Reviewing the code](https://github.com/Expensify/App/blob/main/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified comments were added when the code was not self explanatory
    - [x] I verified any copy / text shown in the product was added in all `src/languages/*` files (if applicable)
    - [x] I verified proper naming convention for platform-specific files was followed (if applicable)
    - [x] I verified [style guidelines](https://github.com/Expensify/App/blob/main/STYLING.md) were followed
- [x] I verified that this PR follows the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/PR_REVIEW_GUIDELINES.md)

### QA Steps
1. Go to Settings
2. Go to a workspace > Connect bank account > Connect manually
3. Press Save without accepting the ToS
4. Observe the error displayed

- [x] Verify that no errors appear in the JS console

### Tested On

- [x] Web
- [x] Mobile Web
- [x] Desktop
- [x] iOS
- [x] Android

#### Web
web

#### Mobile Web
mWeb

#### Desktop
desktop

#### iOS
ios

#### Android
android

### CODE REVIEW

File: src/pages/ReimbursementAccount/BankAccountStep.js [hunk 1]
```diff
@@ -68,6 +68,7 @@ class BankAccountStep extends React.Component {
         this.errorTranslationKeys = {
             routingNumber: 'bankAccount.error.routingNumber',
             accountNumber: 'bankAccount.error.accountNumber',
+            hasAcceptedTerms: 'addDebitCardPage.error.acceptedTerms',
```
Comment (parasharrajat) [inline src/pages/ReimbursementAccount/BankAccountStep.js:71]: Let's move the translations to common place from `addDebitCardPage.error.acceptedTerms` `errors.acceptedTerms`. And refactor other places where this error is used.

**Review by parasharrajat — CHANGES_REQUESTED** [review]

File: src/pages/ReimbursementAccount/BankAccountStep.js (hunk 1 above)
Comment (adeel0202) [reply src/pages/ReimbursementAccount/BankAccountStep.js:71]: @parasharrajat, do you want me to create a new `errors` object at the main level or shall I put `acceptedTerms` in  `common.error` that is already there?

Comment (parasharrajat) [reply src/pages/ReimbursementAccount/BankAccountStep.js:71]: yeah, let's put it in common.error then

**Review by luacmartins — COMMENTED** [review]
Thanks for the changes so far @adeel0202! Let's also include [these](https://github.com/Expensify/App/issues/7839#issuecomment-1056844112) in the refactor

**Comment by adeel0202** [comment]
@luacmartins, I had made the changes @parasharrajat asked for.

**Review by parasharrajat — APPROVED** [review]
LGTM. 

cc: @luacmartins 

#### PR Reviewer Checklist
- [x] I verified the Author pulled `main` before submitting the PR
- [x] I verified the correct issue is linked in the `### Fixed Issues` section above
- [x] I verified testing steps are clear and they cover the changes made in this PR
    - [x] I verified the testing environment is mentioned in the test steps
- [x] I verified testing steps cover success & fail scenarios (if applicable)
- [x] I verified tests pass on **all platforms** & I tested again on all platforms
- [x] I checked that screenshots or videos are included for tests on [all platforms](https://github.com/Expensify/App/blob/main/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I verified there are no console errors related to changes in this PR
- [x] I verified proper code patterns were followed (see [Reviewing the code](https://github.com/Expensify/App/blob/main/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified comments were added when the code was not self explanatory
    - [x] I verified any copy / text shown in the product was added in all `src/languages/*` files (if applicable)
    - [x] I verified proper naming convention for platform-specific files was followed (if applicable)
    - [x] I verified [style guidelines](https://github.com/Expensify/App/blob/main/STYLING.md) were followed
- [x] I verified that this PR follows the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/PR_REVIEW_GUIDELINES.md)

:ribbon: :eyes: :ribbon:  C+ reviewed

**Review by luacmartins — APPROVED** [review]
LGTM!

**Pull request merged**